docs/fundamentals/toc.yml

```diff
@@ -460,7 +460,7 @@ items:
         href: ../standard/base-types/divide-up-strings.md
       - name: Use the StringBuilder class
         href: ../standard/base-types/stringbuilder.md
-      - name: "How to: Perform basic string manipulations"
+      - name: 'How to: Perform basic string manipulations'
```

Comment:
What's the advantage of single quotes over double quotes here?

Reply:
Archive tool again...I'll revert.